src/pkg/services/m365/api/drive.go

```diff
@@ -84,6 +84,26 @@ func (c Drives) GetRootFolder(
 	return root, nil
 }
 
+// TODO: pagination controller needed for completion.
```

Comment:
We should at least increase the number of items we retrieve. I believe the default is 10 which might be too low. Probably same for `GetContainerChildren`.

Reply:
tests are succeeding at the moment, but I do plan to follow up on the pagination.  It's currently pending another PR that'll make implementation super simple.